manifests/s/Stacher/youtube-dl/6.0.14/Stacher.youtube-dl.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Stacher.youtube-dl
+PackageVersion: 6.0.14
+PackageLocale: en-US
+Publisher: StacherIO
+PackageName: UI for youtube-dl

```

```suggestion
PackageName: Stacher
```
This should match Control Panel